core/src/main/resources/db/cgds.sql

```diff
@@ -282,7 +282,7 @@ CREATE TABLE `genetic_profile_samples` (
 CREATE TABLE `sample_profile` (
   `SAMPLE_ID` int(11) NOT NULL,
   `GENETIC_PROFILE_ID` int(11) NOT NULL,
-  UNIQUE KEY `UQ_SAMPLE_ID_GENETIC_PROFILE_ID` (`SAMPLE_ID`,`GENETIC_PROFILE_ID`) COMMENT 'Constraint to allow each sample only once in each profile.',
+  UNIQUE KEY `UQ_SAMPLE_ID_GENETIC_PROFILE_ID` (`SAMPLE_ID`,`GENETIC_PROFILE_ID`), -- Constraint to allow each sample only once in each profile
```

Comment:
Good catch. I think this has been fixed in mysql releases after 5.4, but until we upgrade our production system I think we should continue to comment out our comments on indices. (Before we switch back to actual SQL comments, we should be ready to upgrade our specification of a minimal supported deployment platform to be mysql 5.4 or newer (or whatever the appropriate version is)